.azure-pipelines/internal-release.yml

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
This doesn't look to be referenced anywhere. Is this needed? Or can this just be another deploy stage in the main pipeline?

Reply:
No, it's not needed. Thanks for catching that,